itest/litd_custom_channels_test.go

```diff
@@ -2870,7 +2880,7 @@ func testCustomChannelsHtlcForceClose(_ context.Context, net *NetworkHarness,
 	var (
 		bobHodlInvoices   []assetHodlInvoice
 		aliceHodlInvoices []assetHodlInvoice
-		assetInvoiceAmt   = 100
+		assetInvoiceAmt   = 5_000
```

Comment:
nit: Bit confused on the math for this change causing MPP to get used.

AFAICT, setting `mpp = true` below limits the shard size to 80e6 msats / 80000 sats.

But the minted asset seems to be 1,000,000 `itest-asset-cents`. And it looks like `assetInvoiceAmt` is denominated in cents also. Not sure if decimal display is being set somewhere I can't find.

I also can't find the exchange rate being used here that would probably help clarify this math.

Reply:
Update: I found the rate, from grepping after inspecting the logs, as the config value of `mockoracleassetsperbtc`. Would still be good to call out.

Also this log msg should be `got quote for %d msats`, not sats:

https://github.com/lightninglabs/lightning-terminal/blob/a72e388e5f5d3540d1c90e8b28664bbb7b1c9708/itest/assets_test.go#L1129

So, the quoted sat amount is ~85901 sats, which is over the MPP shard threshold of 80000 sats / 4656 asset units.

Reply:
Ah, yes, that's not super apparent. Added a comment to describe that, which then also makes the use of the `mpp` variable a bit more apparent.

Reply:
I didn't grok this either. Thanks for explaining it @jharveyb.